unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -20,14 +21,26 @@ class TimeTest {
   val now_time_t: time_t = scala.scalanative.posix.time.time(null)
   val epoch: time_t      = 0L
 
+  // some of the tests (the ones that call localtime) need
+  // for the daylight saving time to not be in effect
+  val timeIsDst: Boolean = {
+    Zone { implicit z =>
+      val time_ptr = stackalloc[time_t]
+      !time_ptr = now_time_t
+      val localtime: Ptr[tm] = localtime_r(time_ptr, alloc[tm])
+
+      localtime.tm_isdst != 0
```

Comment:
re: flipping the sense of this block.

To restate what is probably known, but not implemented here.
Posix `tm_isdst` can one of three values: 1, 0, -1.   The -1 means
that  the isdst status is unknown.  As written, the name of the block
belies the test at line 32, because the latter conflates "standard" with
"unknown".

Inverting the sense of the test to `localtime.tm_isdst == 0` and adjusting
the name & use of the block mutatis mutandis isolates a single condition.
That condition is what is ultimately desired/useful.

This may seem picky now, but will aid the next generation of maintainers.

Reply:
Of course, the above ignores the sense of "Standard" in "Irish Standard Time".  There timeIsStandard would, I believe, return false when Irish Standard Time is active ;-)   Probably worth a one line comment.